elasticsearch/README.md

```diff
@@ -35,6 +35,21 @@ Example:
 ```
 $ oc exec -c elasticsearch $POD -- allocate-stale-primary .kibana.02c55f18a892b365bcd1802db9e5c9df39c04674
 ```
+### export-kibana-objects
+Export kibana objects for a given user from their kibana index as a JSON search response
+
+Example:
+```
+$ oc exec -c elasticsearch $POD -- export-kibana-objects admin2
```

Comment:
should you redirect the output e.g. `.... > kib.json` ?

Reply:
yes.